doc/api/perf_hooks.md

```diff
@@ -125,6 +125,18 @@ Creates a new `PerformanceMark` entry in the Performance Timeline. A
 `performanceEntry.duration` is always `0`. Performance marks are used
 to mark specific significant moments in the Performance Timeline.
 
+### performance.maxEntries
+<!-- YAML
+added: REPLACEME
+-->
+
+Value: {number}
```

Comment:
There is a default of 150 set. It should be documented as well.

Reply:
Updated.